docs/documentation/pages/internal/DEVELOPMENT.md

```diff
@@ -46,25 +46,25 @@ Periodicity and timing of updates in the release channels when a minor version c
 There can be several possibilities if a bug is detected in the release progressed to the `Beta` channel (or above):
 1. The bug is in the new functionality:
     1. The release in question contains only the new functionality (that no one uses yet) – in this case, switching versions is canceled (this release will not be advanced to more stable release channels), and the appropriate notification is posted. The bug will be fixed in the subsequent releases.
-    1. The release in question contains other urgently needed changes, or the new functionality is in high demand – in this case, the bug is fixed via hotfix releases.
-1. The bug is in the existing functionality: the bug is fixed via hotfix releases.
+    2. The release in question contains other urgently needed changes, or the new functionality is in high demand – in this case, the bug is fixed via hotfix releases.
+2. The bug is in the existing functionality: the bug is fixed via hotfix releases.
 
 Patch-releases are not stand-alone releases but a set of fixes that are backported to all active releases ASAP (if necessary). Since these changes are backported to all active releases, they should be kept to the minimum necessary!
 
 **If the bug is not urgent, you better be patient and wait until the fix is implemented as part of the standard release process. This is especially true for the `Stable` and `Rock Solid` release channels.**
 
 Periodicity and timing of patch-release updates in the release channels:
 1. The Deckhouse team can change patch-release versions in the `Alpha` channel at any time at its own discretion with any periodicity and without prior warning.
-1. For the `Beta` and `Early Access` channels, the Deckhouse team can switch to a patch-release version at any time at its own discretion but not earlier than two hours after switching to this version in the `Alpha` and `Beta` release channels, accordingly.
-1. For the `Stable` and `Rock Solid` channels, the switching to the patch-release version can be performed on any day:
+2. For the `Beta` and `Early Access` channels, the Deckhouse team can switch to a patch-release version at any time at its own discretion but not earlier than two hours after switching to this version in the `Alpha` and `Beta` release channels, accordingly.
+3. For the `Stable` and `Rock Solid` channels, the switching to the patch-release version can be performed on any day:
    1. `Stable`, `Rock Solid` — not earlier than the next day after the switching to this version in the `Early Access` and `Stable` release channels, accordingly.
-1. These rules may be breached if there is a **real need** to make urgent changes due to a critical bug/vulnerability (note that all actions **must be coordinated with the team leaders**).
+4. These rules may be breached if there is a **real need** to make urgent changes due to a critical bug/vulnerability (note that all actions **must be coordinated with the team leaders**).
```

Comment:
```suggestion
4. These rules may be breached if there is a **real need** to make urgent changes due to a critical bug/vulnerability (do these actions **at your own risk**).
```

Reply:
Sorry, this is not related to the current changes anymore. Not applicable to this PR.